src/pages/authentication/Login.jsx

```diff
@@ -1,11 +1,12 @@
-import { Navbar } from '../../components/navbar/Navbar.jsx'
+import { Navbar } from 'components/navbar/Navbar.jsx'
+import { Footer } from 'components/footer/Footer.jsx'
```

Comment:
You can use barrel export so that you don't have to import again and again it will be done in single line

Reply:
Sure, will do it.
The imports looks good using absolute path, much more readable